apps/builder/app/builder/features/footer/breadcrumbs.tsx

```diff
@@ -8,10 +7,10 @@ import {
   DeprecatedText2,
 } from "@webstudio-is/design-system";
 import {
-  instancesIndexStore,
-  selectedInstanceIdStore,
+  instancesStore,
+  selectedInstanceAddressStore,
 } from "~/shared/nano-states";
-import { getInstanceAncestorsAndSelf } from "~/shared/tree-utils";
+import { getAncestorInstanceAddress } from "~/shared/tree-utils";
```

Comment:
address vs path, if we go with actual path inside then Path word is more accurate

Reply:
We use path a lot of routing and fs in node. Afraid to confuse with naming